src/qt/test/apptests.cpp

```diff
@@ -66,6 +66,7 @@ void AppTests::appTests()
     ECC_Stop(); // Already started by the common test setup, so stop it to avoid interference
     LogInstance().DisconnectTestLogger();
 
+    qRegisterMetaType<interfaces::BlockAndHeaderTipInfo>("interfaces::BlockAndHeaderTipInfo");
```

Comment:
In commit "Reduce cs_main lock accumulation during GUI startup" (ded3edd66c376eed6c9b6a703153d323e47aa4d7)

I guess it'd be nice to register types in a common location in the future to avoid the need to do this twice

Reply:
Agree, suggestion `GUIUtils::registerMetaTypes` or something like that.

Reply:
Yes. Lets do this later when more registration are required/appear.